src/constants.rs

```diff
@@ -15,6 +15,12 @@ lazy_static! {
     pub(crate) static ref PROJECT_NAME: &'static str = env!("CARGO_PKG_NAME");
     pub(crate) static ref PROJECT_VERSION: &'static str = env!("CARGO_PKG_VERSION");
     pub(crate) static ref VERSION_CODENAME: &'static str = include_str!("../CODENAME");
+    pub(crate) static ref CONFIG_PATH: String = {
+        match std::env::var("CARNIVAL_CONFIG_PATH") {
```

Comment:
Maybe having an actual CLI flag for the config path would be better

Reply:
Yeah, but I don't see that as being a requirement for this. Remember that this is a tool separate from Heroic, and I don't see why you wouldn't be able to accommodate a flag instead of an env var there